lib/agoneum/auth/guardian_serializer.ex

```diff
@@ -9,7 +9,7 @@ defmodule Agoneum.GuardianSerializer do
   alias Agoneum.Repo
 
   @spec for_token(any()) :: {:ok, String.t} | {:error, String.t}
-  def for_token(user = %User{}), do: {:ok, "User:#{user.id}"}
```

Comment:
You might want to update @mbramson's https://github.com/mbramson/basic_phoenix_api too